src/web/app/components/question-edit-form/question-edit-form.component.ts

```diff
@@ -168,6 +169,18 @@ export class QuestionEditFormComponent implements OnInit {
 
   ngOnInit(): void {}
 
+  /**
+   * Modifies the model for Mcq question.
```

Comment:
This is still the same unexpected logic as before, you're only moving it from the other component to here. Why don't just change `model.questionDetails` directly?